himbaechel/uarch/gowin/pack.cc

```diff
@@ -1798,6 +1823,15 @@ struct GowinPacker
             bsram_fix_blksel(ci, new_cells);
         }
 
+        // XXX
```

Comment:
assuming this `XXX` is supposed to indicate some kind of hack, can you add a bit of info about what the exact workaround is here, too?

Reply:
Definitely.
Synchronizing CE and OCE helps and the memory works properly in complex systems such as RISC-V emulation and i8080 emulation (with 32K RAM and 16K BSRAM based ROM), but there is no theoretical basis for this fix, so it is a hack.